spam-n-eggs/laravel-mysqlite
# Additional methods

# Description of changes
* Added a number of functions and replacements - See #22 for all functions.
* Added the DIV operator as well.
* Added the HOUR function.
* Cleaned up the composer.json.

# Issues Addressed
* #16
* #22 

# Testing Process
#16 
1. View composer.json.
1. Observe that the information is better.

#22 
1. Execute Test Suites
1. Observe that they all pass - All new methods are covered by unit test.
